survey-runner-queue/ebs.tf

```diff
@@ -0,0 +1,9 @@
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
```

Comment:
I think we need more control over what we're running. The other repo could become compromised which could result in us running the compromised Terraform scripts directly on AWS.